config/routes.rb

```diff
@@ -6,4 +6,6 @@
   end
 
   get '/health', to: 'health#show'
+
+  resource :metrics, only: [:show]
```

Comment:
Would this not make it publicly available? Maybe we need HTTP Basic Auth on this, or is it just for testing for now?